source/renderer/app/components/staking/delegation-center/DelegationCenterHeader.js

```diff
@@ -1,102 +1,184 @@
-import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
-import { SIMPLE_DECIMAL_PLACES_IN_ADA } from '../../../config/numbersConfig';
-import linkNewWindowIcon from '../../../assets/images/link-ic-colored.inline.svg';
-import DonutRing from './DonutRing';
+import { get } from 'lodash';
 import styles from './DelegationCenterHeader.scss';
-import { with2Decimals } from './helpers';
+import CountdownWidget from '../../widgets/CountdownWidget';
+import type { TipInfo } from '../../../api/network/types';
+import delimeterIcon from '../../../assets/images/delimeter.inline.svg';
+import delimeterSlashIcon from '../../../assets/images/delimeter-slash.inline.svg';
 
 const messages = defineMessages({
-  heading: {
-    id: 'staking.delegationCenter.heading',
-    defaultMessage: '!!!Delegation center',
+  epoch: {
+    id: 'staking.delegationCenter.epoch',
+    defaultMessage: '!!!Epoch',
     description: 'Headline for the Delegation center.',
   },
-  descriptionFirstPart: {
-    id: 'staking.delegationCenter.descriptionFirstPart',
-    defaultMessage:
-      '!!!You are currently delegating <b>{adaValue} ADA</b>. This is <b>{percentage}%</b> of ada in your wallets.',
-    description: 'Delegation description-part1 for the Delegation center.',
+  currentSlot: {
+    id: 'staking.delegationCenter.currentSlot',
+    defaultMessage: '!!!Current slot',
+    description: 'Headline for the Delegation center.',
   },
-  descriptionSecondPart: {
-    id: 'staking.delegationCenter.descriptionSecondPart',
-    defaultMessage: '!!!Read',
-    description: 'Delegation description-part2 for the Delegation center.',
+  totalSlots: {
+    id: 'staking.delegationCenter.totalSlots',
+    defaultMessage: '!!!Total slots',
+    description: 'Headline for the Delegation center.',
   },
-  descriptionThirdPart: {
-    id: 'staking.delegationCenter.descriptionThirdPart',
-    defaultMessage: '!!!this article',
-    description: 'Delegation description-part3 for the Delegation center.',
+  headingLeft: {
+    id: 'staking.delegationCenter.headingLeft',
+    defaultMessage: '!!!Cardano epoch {nextEpoch} starts in',
+    description: 'Headline for the Delegation center.',
   },
-  descriptionFourthPart: {
-    id: 'staking.delegationCenter.descriptionFourthPart',
-    defaultMessage: '!!!to learn how to increase the delegated amount.',
-    description: 'Delegation description-part4 for the Delegation center.',
+  headingRight: {
+    id: 'staking.delegationCenter.headingRight',
+    defaultMessage: '!!!Current Cardano epoch',
+    description: 'Headline for the Delegation center.',
+  },
+  description: {
+    id: 'staking.delegationCenter.description',
+    defaultMessage:
+      'Changes to delegation preferences will take effect from the next epoch.',
+    description: 'Delegation description for the Delegation center.',
   },
 });
 
-type Props = { adaValue: BigNumber, percentage: number };
+const TOTAL_SLOTS = 21600;
```

Comment:
@DeeJayElly please move to https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/config/epochsConfig.js

Reply:
Fixed